master/reference/calicoctl/commands/get.md

```diff
@@ -23,16 +23,17 @@ command.
 
 ```
 Usage:
-  calicoctl get ( (<KIND> [<NAME>]) |
+  calicoctl get ( (<KIND> [<NAME...>]) |
                 --filename=<FILENAME>)
                 [--output=<OUTPUT>] [--config=<CONFIG>] [--namespace=<NS>] [--all-namespaces]
 
 Examples:
   # List all policy in default output format.
   calicoctl get policy
 
-  # List a specific policy in YAML format
-  calicoctl get -o yaml policy my-policy-1
+  # List a specific policies in YAML format
```

Comment:
Probably should be "List specific policies" rather than "List a specific policies"

Reply:
Damn, so it is also wrong in the calicoctl code as I copy-pasted it here :-/

Reply:
done